developer-experience/recipes/onboarding-guide-template.md

```diff
@@ -0,0 +1,35 @@
+# Onboarding Guide Template
+
+When developing an onboarding document for a team, it should contain details of engagement scope, team processes, codebase, coding standards, team agreements, software requirements and setup details.
+
+## Overview and Goals
+
+* List a few sentences explaining the high-level summary and the scope of the engagement.
```

Comment:
How about the value prop of the project? Why did the stakeholders fund/support the project in the first place? I think this goes a long way to help align the team on objectives and what success looks like. Thoughts?

Reply:
Good point, will include the value prop and alignment of what success looks like for the project.